src/containers/paint-editor-wrapper.jsx

```diff
@@ -59,7 +73,8 @@ const mapStateToProps = (state, {selectedCostumeIndex}) => {
         name: costume && costume.name,
         rotationCenterX: costume && costume.rotationCenterX,
         rotationCenterY: costume && costume.rotationCenterY,
-        svgId: editingTarget && `${editingTarget}${costume.skinId}`,
+        imageFormat: costume.dataFormat,
```

Comment:
Should this line also check that costume exists first e.g. `costume && costume.dataFormat`?

Reply:
Thanks!